style.css

```diff
@@ -0,0 +1,136 @@
+
+.basic-section {
+    width: 100%;
+    height: 100vh;
+    background-image: url("img/cover.jpg");
+    background-position: center;
+    background-size: cover;
+    opacity: 0.9;
```

Comment:
why did you set the opacity to the whole section?
it applies the opacity to all elements, which is not like on design